### PR TITLE
Julian/create user models

### PR DESCRIPTION
**Pull request by julianrkung**
### Migrations

Set up the initial columns for participant and staff table. Blocked by creation of Paperwork and Case Notes models to create relationships.

### Tests Performed, Edge Cases

No test cases were added.

### Screenshots
:)

CC: @adowski

### CODE REVIEW

File: app/models/participant.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Participant < ApplicationRecord
+  #TODO: Add relationships
```
Comment (adowski) [inline app/models/participant.rb]: Let's try to keep comments (that don't explain/document code) out of master for now! I would delete the TODO & relationship comments.

File: app/models/participant.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  # has_many :paperwork
+  # has_many :case_notes
+  # enum status {r0: 0, r1: 1, r2: 2, studio: 3}
```
Comment (adowski) [inline app/models/participant.rb]: This enum can be uncommented! I also think you're missing a colon after status - it should be `status:`. 

Also let me know when you have time and we can go over how to use `rails console` to check that the models/enums are working correctly :)

File: app/models/staff.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Staff < ApplicationRecord
+  #TODO: Add relationships
```
Comment (adowski) [inline app/models/staff.rb]: Same as above w/ the comments
